theseus/utils/examples/__init__.py

```diff
@@ -3,6 +3,11 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from .bundle_adjustment import (
+    LocalizationDataset,
+    LocalizationSample,
+    ReprojectionError,
```

Comment:
`ReprojectionError` (-> `Reprojection`) could go to `embodied/camera` as this cost function would be more widely useful.